INSTALL.md

```diff
@@ -0,0 +1,29 @@
+
+# Sourcery VS Code Client
+
+Sourcery is an AI-powered coding assistant which helps you write better Python
+code faster. It works by providing refactoring suggestions on the fly that you
+can instantly integrate into your code.
+
+This extension is currently in beta - please raise any problems or feedback as
+issues.
+
+# Installation
+
+1. Go to https://sourcery.ai/download to get a free token and copy it to your 
+   clipboard.
+2. Enter the token into the box above, or search for `sourcery` in the 
+   VS Code settings and enter the token into the `Sourcery Token` field.
+
+# How it works
+
+Sourcery scans the Python file that you have open in your editor. Refactoring 
+suggestions are underlined and listed in the Problems window. 
+
+You can hover over a suggestion to see a diff of the changes.
+
+To implement the suggestion use the Quick Fix from either the Problems window
+or the lightbulb in the code editor.
```

Comment:
- The lightbulb is only in the Problems window
- Lightbulb appears on all problems not just sourcery ones

Reply:
Presume the README.md needs updating too

Reply:
addressed in next version of binary